### PR TITLE
Add example's README

### PR DESCRIPTION
**Pull request by ftersin**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    Exapmles show terraform users how to create vkcs objects with various features.
+    Users may just look inside examples or run them to get trained.
+
+- Release/development automatization
+
+    Examples are used by canary tests on vkcs side to verify the latests terraform
+    and vkcs provider over vkcs cloud.
+
+    Also exapmles are used by developers manually or via CI (**in construction**)
```
Comment (schirevko) [inline examples/README.md:15]: examples

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    Some examples are planned to be built inside documentation on guide/resource pages
+    by docomentation generator.
+
+## Running an example
+
+- Choose an example
+- Create a folder on a local host
+- Set up vkcs terraform provider in the folder using instruction on your account page on vkcs site.
+- Download all files of the example into the folder.
+- Run `terraform apply`. This will deploy the example onto vkcs.
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md:29]: `terraform init` should be described in installation instruction. And it is so at the moment: https://mcs.mail.ru/app/mcs2253251547/project/terraform

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+An example must be consistent. i.e. it must describe all required resources/datasources
+to run the example in a separate folder.
+
+An example must contain `main.tf` file.
+
+`main.tf` file must contain enough resources to demonstrate an idea of the example itself.
+E.g. `main.tf` of compute resource example must contain compute resource.
+
+`main.tf` file must **not** contain resources/datasource do not direcly relate to an idea
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md:44]: relate is active verb in this case. probably just add 'wich' before 'do'?

File: examples/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Examples
+
+## Audience of examples
+
+- Users
+
+    Exapmles show terraform users how to create vkcs objects with various features.
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md]: done

File: examples/README.md (hunk 1 above)
Comment (ftersin) [reply examples/README.md]: done

File: examples/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    Examples are used by canary tests on vkcs side to verify the latests terraform
+    and vkcs provider over vkcs cloud.
+
+    Also exapmles are used by developers manually or via CI (**in construction**)
+    to verify changes.
+
+- Documentation generator (**in construction**)
+
+    Some examples are planned to be built inside documentation on guide/resource pages
+    by docomentation generator.
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md]: done

File: examples/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md:30]: done except above

File: examples/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+An example must contain `main.tf` file.
+
+`main.tf` file must contain enough resources to demonstrate an idea of the example itself.
+E.g. `main.tf` of compute resource example must contain compute resource.
+
+`main.tf` file must **not** contain resources/datasource do not direcly relate to an idea
+of the example itself.
+E.g. `main.tf` of compute resource example must not contain network resources.
+
+Additional resources/datasources of an example must be specified in additoonal file of
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md]: done

File: examples/README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+the example. At the moment all such examples use the single `base.tf` file for this purpuse,
```
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md]: done

File: examples/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ftersin) [reply examples/README.md]: done

**Review by schirevko — APPROVED** [review]

**Pull request merged**